packages/patternfly-4/react-table/src/components/Table/utils/transformers.tsx

```diff
@@ -0,0 +1,40 @@
+export { selectable } from './decorators/selectable';
+export { sortable } from './decorators/sortable';
+export { cellActions } from './decorators/cellActions';
+export { cellWidth } from './decorators/cellWidth';
+export { textCenter } from './decorators/textCenter';
+export { collapsible, expandedRow, expandable } from './decorators/collapsible';
+export { compoundExpand } from './decorators/compoundExpand';
+export { headerCol } from './decorators/headerCol';
+export { classNames, Visibility } from './decorators/classNames';
+
+import { IFormatterValueType, IExtra } from '../Table';
+
+const emptyTD = () => ({
+  scope: '',
+  component: 'td'
+});
+emptyTD.prototype.name = 'emptyTD';
```

Comment:
extending function prototype like this is breaking downstream... will find a different workaround. This mainly just to preserve existing tests using the prototype `name` on `Function`, however Typescript does not support this.

Reply:
it seems that some dependency updates on my local have resolved this. Should not be necessary to extend `prototype`.